include/odp/api/spec/pool.h

```diff
@@ -92,6 +92,12 @@ typedef struct odp_pool_capability_t {
 		 * memory size for the pool. */
 		uint32_t max_num;
 
+		/** Maximum packet data alignment in bytes
+		 *
+		 * This is the maximum value of packet pool alignment
+		 * (pkt.align) parameter. */
```

Comment:
Shouldn't `*/` be on the next line?

Reply:
This style is has been used on this capa struct. So, used the same here as well.